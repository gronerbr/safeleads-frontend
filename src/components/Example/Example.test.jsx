import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Example from './Example';

test('renders without crashing', () => {
  const component = renderer.create(
    <MemoryRouter location="someLocation" context={{}}>
      <Example />
    </MemoryRouter>);

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
