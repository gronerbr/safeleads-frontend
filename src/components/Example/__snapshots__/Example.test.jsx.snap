// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <p>
    It
    '
    s an example of React component. You can navigate to  
    <a
      href="/counter"
      onClick={[Function]}
    >
      Counter
    </a>
      to see another page.
  </p>
</div>
`;
